plugin/trino-mongodb/src/main/java/io/trino/plugin/mongodb/MongoSession.java

```diff
@@ -359,6 +371,9 @@ private MongoTable loadTableSchema(SchemaTableName schemaTableName)
         RemoteTableName remoteSchemaTableName = toRemoteSchemaTableName(schemaTableName);
         String remoteSchemaName = remoteSchemaTableName.getDatabaseName();
         String remoteTableName = remoteSchemaTableName.getCollectionName();
+        if (!collectionExists(client.getDatabase(remoteSchemaName), remoteTableName)) {
+            throw new TableNotFoundException(schemaTableName, format("Table '%s.%s' not found", schemaTableName.getSchemaName(), schemaTableName.getTableName()), null);
```

Comment:
Remove the 2nd & 3rd arguments.

Reply:
applied. Since `getTableMetadata` uses this method, I referred to it. Would it be good to fix the `TableNotFoundException` generation method of `getTableMetadata` later?